.github/workflows/schedule-posts.txt

```diff
@@ -0,0 +1,39 @@
+name: Publish posts scheduled for today
+
+on:
+  schedule:
+    # Run every day at 23:30 UTC or manually run
+    - cron: "30 23 * * *"
+  workflow_dispatch:
+
+jobs:
+  publish_drafts:
```

Comment:
Last change: maybe rename from `publish_drafts` to `publish_scheduled`, since we don't mess with the `_drafts/` dir.